geoh5py/objects/grid2d.py

```diff
@@ -410,3 +338,13 @@ def to_geoimage(
         :return: a new georeferenced :obj:`geoh5py.objects.geo_image.GeoImage`.
         """
         return self.converter.to_geoimage(self, keys, mode=mode, **geoimage_kwargs)
+
+    @staticmethod
+    def validate_count(value: int, axis: str) -> np.int32:
+        """
+        Validate and format type of count value.
+        """
+        if not isinstance(value, (np.integer, int)):
```

Comment:
we should validate if count is positive too

Reply:
Greater than 1, yes